client/hosting/server-settings/components/cache-card/index.tsx

```diff
@@ -126,57 +82,109 @@ export const CacheCard = ( {
 						},
 					} ) }
 				</HostingCardDescription>
-				<ToggleContainer>
-					{ getEdgeCacheInitialLoading ? (
-						<EdgeCacheLoadingPlaceholder />
-					) : (
-						<>
-							<ToggleLabel>{ translate( 'Global edge cache' ) }</ToggleLabel>
-							<ToggleControl
-								disabled={
-									clearEdgeCacheLoading ||
-									getEdgeCacheLoading ||
-									! isEdgeCacheEligible ||
-									isEdgeCacheMutating
-								}
-								checked={ isEdgeCacheActive && isEdgeCacheEligible }
-								onChange={ ( active ) => {
-									recordTracksEvent(
-										active
-											? 'calypso_hosting_configuration_edge_cache_enable'
-											: 'calypso_hosting_configuration_edge_cache_disable',
-										{
-											site_id: siteId,
-										}
-									);
-									setEdgeCache( siteId, active );
-								} }
-								label={ edgeCacheToggleDescription }
-							/>
-							<Hr />
-						</>
-					) }
-				</ToggleContainer>
-				{ getClearCacheContent() }
-			</CardBody>
+				<Button
+					onClick={ handleClearCache }
+					busy={ isClearingCache }
+					disabled={
+						disabled ||
+						isClearingCache ||
+						shouldRateLimitCacheClear ||
+						getEdgeCacheLoading ||
+						isEdgeCacheMutating
+					}
+					className="performance-optimization__button"
+				>
+					<span>{ translate( 'Clear all caches' ) }</span>
+				</Button>
+				<div className="performance-optimization__nb">
+					{ translate( 'Clearing the cache may temporarily make your site less responsive.' ) }
+				</div>
+				{ shouldRateLimitCacheClear && (
+					<div className="performance-optimization__nb">
+						{ translate( 'You cleared the cache recently. Please wait a minute and try again.' ) }
+					</div>
+				) }
+			</div>
+
+			<div className="performance-optimization__hr"></div>
+
+			<div className="performance-optimization__global-edge-cache-block">
+				{ getEdgeCacheInitialLoading ? (
+					<EdgeCacheLoadingPlaceholder />
+				) : (
+					<>
+						<div className="performance-optimization__subtitle">
+							{ translate( 'Global edge cache' ) }
+						</div>
+						<ToggleControl
+							disabled={
+								clearEdgeCacheLoading ||
+								getEdgeCacheLoading ||
+								! isEdgeCacheEligible ||
+								isEdgeCacheMutating
+							}
+							checked={ isEdgeCacheActive && isEdgeCacheEligible }
+							onChange={ ( active ) => {
+								recordTracksEvent(
+									active
+										? 'calypso_hosting_configuration_edge_cache_enable'
+										: 'calypso_hosting_configuration_edge_cache_disable',
+									{
+										site_id: siteId,
+									}
+								);
+								setEdgeCache( siteId, active );
+							} }
+							label={ edgeCacheToggleDescription }
+						/>
+						{ config.isEnabled( 'hosting-server-settings-enhancements' ) && (
+							<Button onClick={ () => {} } className="performance-optimization__button">
+								<span>{ translate( 'Clear edge cache' ) }</span>
+							</Button>
```

Comment:
```suggestion
							<Button
							    disabled={ something… }
								onClick={ () => {
									clearEdgeCache();
								} }
								className="performance-optimization__button"
							>
								<span>{ translate( 'Clear edge cache' ) }</span>
							</Button>
```
We already have access to the necessary function for making the button do something when clicked.

We should also take into account the `disabled` logic. I'm actually not sure if we need to rate limit both these buttons, but maybe it's an easy starting point to disable both new buttons under the same conditions as the `Clear all caches` button.

Lastly, we should hide this button when edge cache is disabled.